test/unittestsuite.go

```diff
@@ -0,0 +1,34 @@
+package testutils
```

Comment:
I thought you said you want this Suite in a separate PR? :)

Reply:
Yes. And there is already one. Could be that there is some mixup here. Will check.

Reply:
Ok, somehow that testsuite file ended up in this PR, all other changes for the suites are where they should be. Removed it in 97c2f18